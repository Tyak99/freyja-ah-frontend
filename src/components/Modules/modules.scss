.m-container {
  width: 100%;
}
.m-container div .card{
  width: 33.333%;
  margin:0;
  position: unset;
  width: 350px;
}