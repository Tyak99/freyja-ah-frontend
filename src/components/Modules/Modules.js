/* eslint-disable react/prefer-stateless-function */
import React from 'react';
import {
  CardBody, CardTitle, Row, Col, CardSubtitle, CardText, Button,
} from 'reactstrap';
import Card from '../Card/Card';
import './modules.scss';

const Module = () => (
  <div className="m-container cd-view view">
    <div className="m-intro">
      <h3>Modules</h3>
      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
    </div>
    <Row>
      <Col sm={{ size: 'auto' }}>
        <Card>
          <CardBody className="m-c-title">
            <CardTitle>Module 1</CardTitle>
            <CardSubtitle>Description</CardSubtitle>
            <CardText>
              Lorem ipsum dolor sit amet, consectetur adipiscing elit.
            </CardText>
            <CardSubtitle>Introduction</CardSubtitle>
            <CardText>
              Lorem ipsum dolor sit amet, consectetur adipiscing elit.
            </CardText>
          </CardBody>
        </Card>
      </Col>
      <Col sm={{ size: 'auto' }}>
        <Card>
          <CardBody className="m-c-title">
            <CardTitle>Module 2</CardTitle>
            <CardSubtitle>Description</CardSubtitle>
            <CardText>
              Lorem ipsum dolor sit amet, consectetur adipiscing elit.
            </CardText>
            <CardSubtitle>Introduction</CardSubtitle>
            <CardText>
              Lorem ipsum dolor sit amet, consectetur adipiscing elit.
            </CardText>
          </CardBody>
        </Card>
      </Col>
      <Col sm={{ size: 'auto' }}>
        <Card>
          <CardBody className="m-c-title">
            <CardTitle>Module 3</CardTitle>
            <CardSubtitle>Description</CardSubtitle>
            <CardText>
              Lorem ipsum dolor sit amet, consectetur adipiscing elit.
            </CardText>
            <CardSubtitle>Introduction</CardSubtitle>
            <CardText>
              Lorem ipsum dolor sit amet, consectetur adipiscing elit.
            </CardText>
          </CardBody>
        </Card>
      </Col>
    </Row>
    <div className="take-course">
      <Button>
        Take Course
      </Button>
    </div>
  </div>
);


export default Module;
