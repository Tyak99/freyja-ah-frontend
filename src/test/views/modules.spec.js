import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Modules from '../../views/Modules/modules';

const setUp = (props = {}) => {
  const component = shallow(<Modules {...props} />);
  return component;
};
describe('Modules component', () => {
  let component;
  beforeEach(() => {
    component = setUp();
  });
  it('should render without errors', () => {
    expect(toJson(component)).toMatchSnapshot();
  });
  it('should render children', () => {
    const wrapper = component.find('h2');
    expect(wrapper.exists()).toBe(true);
  });
});
