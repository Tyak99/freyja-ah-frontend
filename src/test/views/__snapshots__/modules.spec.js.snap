// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modules component should render without errors 1`] = `
<div>
  <h2>
     This is the Module pPage
  </h2>
  <Link
    to="/login"
  >
     Go to Login Page 
  </Link>
  <br />
  <Link
    to="/signup"
  >
     Go to Signup Page 
  </Link>
</div>
`;
